package.json

```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "description": "Tell Slack you're working from home today 🏡",
   "repository": {
-	  "type": "git",
```

Comment:
Oh, you know what, I think I forgot add a rule to the `.editorconfig` for `package.json` as a 2-space file. Would you mind doing that?